desktop/onionshare/resources/locale/ga.json

```diff
@@ -137,5 +137,121 @@
     "gui_chat_stop_server": "Stop an freastalaí comhrá",
     "gui_chat_start_server": "Tosaigh an freastalaí comhrá",
     "gui_file_selection_remove_all": "Bain Uile",
-    "gui_remove": "Bain"
+    "gui_remove": "Bain",
+    "gui_settings_tor_bridges_label": "",
+    "gui_settings_version_label": "",
+    "gui_settings_stop_active_tabs_label": "",
+    "gui_settings_help_label": "",
+    "gui_close_tab_warning_close": "",
+    "gui_quit_warning_title": "",
+    "gui_color_mode_changed_notice": "",
+    "gui_main_page_chat_button": "",
+    "mode_settings_autostop_timer_checkbox": "",
+    "gui_settings_theme_dark": "",
+    "gui_autostop_timer_cant_be_earlier_than_autostart_timer": "",
+    "gui_share_url_public_description": "",
+    "gui_receive_url_public_description": "",
+    "gui_chat_url_public_description": "",
```

Comment:
Here there are some untranslated strings.

Do you want untranslated strings to appear like this?

We can:

- Put the English string when there is no translation
- remove the string completely (the whole line, including the key)
- Leave it as is (lots of changes on every file, even when there is no translation activity)

What do you think is best?

Reply:
Untranslated strings get displayed in English, but you shouldn't have to actually insert the English string here. See here: https://github.com/onionshare/onionshare/blob/main/desktop/onionshare/strings.py#L53

Basically, if a string is blank, then it automatically defaults to displaying English.

Reply:
> Untranslated strings get displayed in English, but you shouldn't have to actually insert the English string here. See here: https://github.com/onionshare/onionshare/blob/main/desktop/onionshare/strings.py#L53
> 
> Basically, if a string is blank, then it automatically defaults to displaying English.

OK, I want to **remove the whole line** when the string is not translated, then. This will make the translations less noisy because they will not be updated with empty translations if they are not active.

I understand that it will be ok, because is covered by https://github.com/onionshare/onionshare/blob/a01315295eaf5b2e607ebbddeca04ed034bbbfa4/desktop/onionshare/strings.py#L48  (' and s in translations[current_locale]' ).

I will change the configuration in weblate, and we can see if next weblate merge request passes the tests. If not, I will return to the current setup.